registration/src/routes/+page.server.ts

```diff
@@ -155,8 +156,11 @@ export const actions: Actions = {
 
 				return fail(400, { phone, phone_taken: true });
 			}
+			let token = DEMO_TOKEN;
 
-			const token = await send(phone);
+			if (env.DEMO_MODE !== 'false') {
+				token = await send(phone);
```

Comment:
If `DEMO_MODE == true` send SMS ? Should be `!== true` ?

Reply:
yes correct thanks